File: linear_search.py
def linear_search(num,count,x):
    flag=0
    for j in range (0,count):
        if num[j]==x:
            print(f"Number {x} found at index {j}")
            break
        else:
            flag=1

    if flag==1:
        print(f"Number {x} not found")

count=int(input("Enter the number of elements in the array: "))

num=[]
for i in range (0,count):
    y=int(input(":"))
    num.append(y)
    i+=1

x=int(input("Enter the number to search: "))

linear_search(num,count,x)

File: linear_search.c
#include<stdio.h>

int linear_search(int x, int count, int num[])
{
    for (int j=0; j<count; j++)
    {
        if (num[j]==x)
        {
            return j;
        }
    }

    return 1;
}

int main()
{
    int count,x;
    int num[count];
    printf("Enter the number of elements in the array: ");
    scanf("%i",&count);

    for (int i=0; i<count; i++)
    {
        num[i]=i;        
    }

    printf("Enter the number to search: ");
    scanf("%i\n",&x);

    int ret=linear_search(x,count,num);

    (ret==1)?printf("Number not found"):printf("Number found at index %i",ret);

    return 0;
}